discovery-provider/plugins/network-monitoring/src/content/queries.ts

```diff
@@ -287,13 +287,17 @@ export const getUserCounts = async (run_id: number, spid: number): Promise<[numb
     })
 
     const userCounts = (userCountsResp as {
-        spid: number,
-        primary_count: number,
-        secondary1_count: number,
-        secondary2_count: number,
-    }[])[0] || { primary_count: 0, secondary1_count: 0, secondary2_count: 0 }
-
-    return [userCounts.primary_count, userCounts.secondary1_count, userCounts.secondary2_count]
+        spid: string,
+        primary_count: string,
+        secondary1_count: string,
+        secondary2_count: string,
+    }[])[0] || { primary_count: '0', secondary1_count: '0', secondary2_count: '0' }
+
+    return [
+        parseInt(userCounts.primary_count), 
+        parseInt(userCounts.secondary1_count), 
+        parseInt(userCounts.secondary2_count),
+    ]
 }
```

Comment:
just changing defaults to strings?

Reply:
This was a bug - they were supposed to be numbers but sequelize was returning them as strings., Typescript doesn't catch this